domains/dhruvakalur.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "My Personal EMAIL",
+  "owner": {
+    "username": "dhruvakalur",
+    "email": "dhruvakalur9@gmail.com",
```

Comment:
Remove trailing `,`

```suggestion
    "email": "dhruvakalur9@gmail.com"
```

Reply:
the server host i use says i must first set A record to their IP before proxying

45.90.12.29:7284 : preview